docs/changes.rst

```diff
@@ -235,6 +235,8 @@ New features:
   Solves the potential AES-CTR mode counter management issues of the legacy crypto.
 - init: --key-algorithm=argon2 (new default KDF, older pbkdf2 also still available)
   borg key change-passphrase / change-location keeps the key algorithm unchanged.
+- key change-algorithm: to upgrade existing repos to argon2 or downgrade to pbkdf2.
+  We recommend you to upgrade unless you have to keep the repo compatible with older versions of borg.
```

Comment:
hmm, just noticed that we maybe rather should say keys (not repos).

2 places.